### PR TITLE
Module 1 exercise

### PR DESCRIPTION
**Pull request by The-Build-Master**
Module 1 exercise

Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

**Comment by The-Build-Master** [comment]
I hope I have got this right

File: todo_app/app.py [hunk 1]
```diff
@@ -1,11 +1,25 @@
 from flask import Flask
+from flask import request
+from flask import redirect
+from flask import url_for
+from flask import render_template
```
Comment (JoshAdamPowell) [inline todo_app/app.py:5]: Minor thing, but you can import these all on the same line:
`from flask import Flask, request, redirect, url_for, render_template`

File: todo_app/app.py [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
 
 from todo_app.flask_config import Config
 
+from todo_app.data.session_items import get_items
```
Comment (JoshAdamPowell) [inline todo_app/app.py:9]: same thing about importing in one line

**Review by JoshAdamPowell — APPROVED** [review]
I've left a tiny comment about how your code could be improved but feel free to go ahead and merge anyway - good work!

**Pull request merged**